### PR TITLE
Scale 414 on aws

### PR DESCRIPTION
**Pull request by paigerube14**
Adding scale with small size on aws 414

### CODE REVIEW

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.14-ocp-qe-perfscale-ci-tests-kube-burner periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.14-ocp-qe-perfscale-node-density-heavy-aws periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.14-ocp-qe-perfscale-router-perf-aws periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.14-ocp-qe-perfscale-ci-tests-net-perf

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.14-ocp-qe-perfscale-ci-tests-kube-burner periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.14-ocp-qe-perfscale-router-perf-aws

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.14-ocp-qe-perfscale-node-density-heavy-aws

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.14-ocp-qe-perfscale-node-density-heavy-aws  periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.14-ocp-qe-perfscale-router-perf-aws periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.14-ocp-qe-perfscale-ci-tests-net-perf

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.14-ocp-qe-perfscale-node-density-heavy-aws periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.14-ocp-qe-perfscale-router-perf-aws periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.14-ocp-qe-perfscale-ci-tests-net-perf

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.14-ocp-qe-perfscale-ci-tests-kube-burner

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.14-ocp-qe-perfscale-ci-tests-kube-burner

**Comment by paigerube14** [comment]
@qiliRedHat @svetsa-rh PTAL

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.14-ocp-qe-perfscale-ci-tests-kube-burner

**Comment by paigerube14** [comment]
@svetsa-rh PTAL

**Comment by svetsa-rh** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.14-ocp-qe-perfscale-ci-tests-net-perf periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.14-ocp-qe-perfscale-node-density-heavy-aws periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.14-ocp-qe-perfscale-router-perf-aws

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14.yaml [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      WORKER_REPLICA_COUNT: "20"
+    post:
+    - chain: ipi-aws-post
+    pre:
+    - chain: ipi-aws-pre
+    - ref: workers-scale
+    workflow: openshift-qe-ocp-qe-perfscale-ci-tests
```
Comment (svetsa-rh) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14.yaml:36]: @paigerube14 Do we want to use this workflow for all perfscale-ci-tests or instead list all tests at this high level config?
The reason I ask is, if any of the tests in that workflow fail, the remaining tests won't run. But, if we list all tests we gurantee running all tests independent of each other's test result.

Comment (paigerube14) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14.yaml:36]: Will it run the rest of the tests if one fails? I've never seen that occur. I've only seen if one test fails it'll go to the post

Comment (svetsa-rh) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14.yaml:36]: If each test is mentioned separately, they run independently.

Comment (svetsa-rh) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14.yaml:36]: Another option I can think of is, changing `openshift-qe-ocp-qe-perfscale-ci-tests` to chain from workflow. That will also allow all tests to run independent of each other's result. And that will also allow us to keep high level config clean. Something worth trying IMO.

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.14-ocp-qe-perfscale-ci-tests-net-perf periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.14-ocp-qe-perfscale-node-density-heavy-aws periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.14-ocp-qe-perfscale-router-perf-aws periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.14-ocp-qe-perfscale-ci-tests-kube-burner

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.14-ocp-qe-perfscale-ci-tests-net-perf periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.14-ocp-qe-perfscale-node-density-heavy-aws

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.14-ocp-qe-perfscale-ci-tests-net-perf

**Comment by paigerube14** [comment]
hitting an issue locally when trying to use the write script in a chain with each workflow. Going to do something like [this](https://github.com/openshift/release/tree/b87dd45f7b093189ae63a31df5f0be69b3a4ea0d/ci-operator/step-registry/single-node/add-worker/3) to be able to use the same base script but with different names ot get around it

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.14-ocp-qe-perfscale-ci-tests-net-perf

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.14-ocp-qe-perfscale-ci-tests-net-perf

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14.yaml (hunk 1 above)
Comment (paigerube14) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14.yaml:36]: think we should investigate this in a different PR. want to try to get this in

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.14-ocp-qe-perfscale-node-density-heavy-aws

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.14-ocp-qe-perfscale-node-density-heavy-aws

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.14-ocp-qe-perfscale-node-density-heavy-aws

**Comment by paigerube14** [comment]
I did find that the prometheus pods were terminating and the 2 nodes they were on were not ready. Lowering pods_per_node count and seeing how it does 


```
here are 197 running in node ip-****.ap-northeast-1.compute.internal
There are 215 running in node ip-****.ap-northeast-1.compute.internal
There are 213 running in node ip-****.ap-northeast-1.compute.internal
There are 194 running in node ip-****.ap-northeast-1.compute.internal
There are 188 running in node ip-******13.ap-northeast-1.compute.internal
There are 250 running in node ip-******102.ap-northeast-1.compute.internal
There are 196 running in node ip-******143.ap-northeast-1.compute.internal
There are 286 running in node ip-******72.ap-northeast-1.compute.internal
There are 205 running in node ip-******249.ap-northeast-1.compute.internal
There are 250 running in node ip-******224.ap-northeast-1.compute.internal
There are 234 running in node ip-******40.ap-northeast-1.compute.internal
There are 188 running in node ip-******137.ap-northeast-1.compute.internal
There are 206 running in node ip-******18.ap-northeast-1.compute.internal
There are 201 running in node ip-******213.ap-northeast-1.compute.internal
There are 250 running in node ip-******163.ap-northeast-1.compute.internal
There are 250 running in node ip-******144.ap-northeast-1.compute.internal
There are 189 running in node ip-******214.ap-northeast-1.compute.internal
There are 250 running in node ip-******107.ap-northeast-1.compute.internal
There are 251 running in node ip-******27.ap-northeast-1.compute.internal
There are 250 running in node ip-******215.ap-northeast-1.compute.internal


% oc get nodes
NAME                                              STATUS     ROLES                  AGE   VERSION
ip-******37.ap-northeast-1.compute.internal    Ready      control-plane,master   79m   v1.27.2+55f2dbe
ip-****.ap-northeast-1.compute.internal   Ready      worker                 53m   v1.27.2+55f2dbe
ip-****.ap-northeast-1.compute.internal   Ready      worker                 53m   v1.27.2+55f2dbe
ip-****.ap-northeast-1.compute.internal    Ready      worker                 53m   v1.27.2+55f2dbe
ip-****.ap-northeast-1.compute.internal   Ready      worker                 66m   v1.27.2+55f2dbe
ip-******13.ap-northeast-1.compute.internal    Ready      worker                 68m   v1.27.2+55f2dbe
ip-******102.ap-northeast-1.compute.internal   Ready      worker                 53m   v1.27.2+55f2dbe
ip-******143.ap-northeast-1.compute.internal   Ready      worker                 53m   v1.27.2+55f2dbe
ip-******72.ap-northeast-1.compute.internal    NotReady   worker                 53m   v1.27.2+55f2dbe
ip-******249.ap-northeast-1.compute.internal   Ready      worker                 53m   v1.27.2+55f2dbe
ip-******224.ap-northeast-1.compute.internal   Ready      worker                 52m   v1.27.2+55f2dbe
ip-******102.ap-northeast-1.compute.internal   Ready      control-plane,master   80m   v1.27.2+55f2dbe
ip-******40.ap-northeast-1.compute.internal    Ready      worker                 53m   v1.27.2+55f2dbe
ip-******137.ap-northeast-1.compute.internal   Ready      worker                 53m   v1.27.2+55f2dbe
ip-******18.ap-northeast-1.compute.internal    Ready      worker                 53m   v1.27.2+55f2dbe
ip-******180.ap-northeast-1.compute.internal   Ready      control-plane,master   79m   v1.27.2+55f2dbe
ip-******213.ap-northeast-1.compute.internal   Ready      worker                 53m   v1.27.2+55f2dbe
ip-******163.ap-northeast-1.compute.internal   Ready      worker                 53m   v1.27.2+55f2dbe
ip-******144.ap-northeast-1.compute.internal   Ready      worker                 53m   v1.27.2+55f2dbe
ip-******214.ap-northeast-1.compute.internal   Ready      worker                 53m   v1.27.2+55f2dbe
ip-******107.ap-northeast-1.compute.internal   NotReady   worker                 52m   v1.27.2+55f2dbe
ip-******27.ap-northeast-1.compute.internal    Ready      worker                 68m   v1.27.2+55f2dbe
ip-******215.ap-northeast-1.compute.internal   Ready      worker                 53m   v1.27.2+55f2dbe

prubenda@prubenda1-mac ~ % oc get pods -n openshift-monitoring -o wide | grep "ip-******107.ap-northeast-1.compute.internal|ip-******72.ap-northeast-1.compute.internal"
prubenda@prubenda1-mac ~ % oc get pods -n openshift-monitoring -o wide | grep "prom"                                                                                          
prometheus-adapter-797bf7dfc4-4frbq                      1/1     Running       0          36m     10.128.6.232    ip-******40.ap-northeast-1.compute.internal    <none>           <none>
prometheus-adapter-797bf7dfc4-rk77v                      1/1     Running       0          67m     10.128.2.15     ip-******13.ap-northeast-1.compute.internal    <none>           <none>
prometheus-k8s-0                                         6/6     Terminating   0          9m7s    10.130.10.253   ip-******107.ap-northeast-1.compute.internal   <none>           <none>
prometheus-k8s-1                                         6/6     Terminating   0          7m28s   10.131.2.250    ip-******72.ap-northeast-1.compute.internal    <none>           <none>
prometheus-operator-5c99f558f7-lcgvg                     2/2     Running       0          67m     10.130.0.46     ip-******180.ap-northeast-1.compute.internal   <none>           <none>
prometheus-operator-admission-webhook-7f4cd95bdb-bvsfz   1/1     Running       0          36m     10.128.6.231    ip-******40.ap-northeast-1.compute.internal    <none>           <none>
prometheus-operator-admission-webhook-7f4cd95bdb-l6g7d   1/1     Running       0          76m     10.128.2.9      ip-******13.ap-northeast-1.compute.internal    <none>           <none>  ~

 % oc get pods -n a614cb4a-08b3-4d79-b16b-db7f77a0c48f | wc -l      
    4539

prubenda@prubenda1-mac ~ % oc get pods -n a614cb4a-08b3-4d79-b16b-db7f77a0c48f | grep Pending |wc -l
     418

```

**Comment by paigerube14** [comment]
@svetsa-rh @qiliRedHat PTAL, past workloads you can see all pass in https://prow.ci.openshift.org/pr-history/?org=openshift&repo=release&pr=39751

**Comment by qiliRedHat** [comment]
@paigerube14 @svetsa-rh I didn't realize my /lgtm directly merged this pr. I wanted to add my own and leave to Sharada to do her final review. Next time I think we can do it like this: pr owner to tag all people who is expected to provide the /lgtm and the early reviewer just add 'look good to me' text and leave it to the last reviewer to add /lgtm, sounds ok? Let me know your thoughts.

**Pull request merged**